src/Units/UnitModel.java

```diff
@@ -35,38 +35,62 @@
   public int maxFuel;
   public int idleFuelBurn;
   public int movePower;
+  public int vision;
+  public boolean piercingVision = false;
```

Comment:
What is `piercingVision`?

Reply:
Goes through cover, like forests/reefs.
Admittedly, it could probably use a comment.